rust/src/lib/route.rs

```diff
@@ -77,6 +79,34 @@ impl Routes {
                     error!("{}", e);
                     return Err(e);
                 }
+                if let Some(dst) = route.destination.as_ref() {
+                    let dst_str = dst.as_str();
+                    if !is_ipv6_addr(dst_str) {
+                        let mut is_invalid_dst = false;
+                        if dst_str == "0.0.0.0" {
+                            is_invalid_dst = true;
+                        } else if dst_str.contains('/') {
+                            let ip_net: Vec<&str> =
+                                dst_str.split('/').collect();
+                            let ip_addr = Ipv4Addr::from_str(ip_net[0]);
+                            let prefix = ip_net[1].parse::<i32>().unwrap_or(32);
+                            if ip_addr?.octets()[0] == 0 && prefix >= 8 {
```

Comment:
Normally, we do not place `?` in the middle of statement. Please try `let ip_addr = Ipv4Addr::from_str(ip_net[0])?;"

Reply:
I updated. thanks.